Automattic/woocommerce-payments
# RPP: Create a payment service class

Note: RPP = Re-engineering Payment Process. 

Fixes #6674

Relevant links: 

- https://betadownload.jetpack.me/data/woocommerce-payments/rpp_6674-create-payment-service-class/woocommerce-payments-dev.zip
- https://jurassic.ninja/wp-json/jurassic.ninja/jetpack-beta/plugins/woocommerce-payments/branches


#### Changes proposed in this Pull Request

- Add a new `src` folder, and the new `PaymentService` class.









#### Testing instructions

- Ensure no CI tests fail. 
- Check the build that can be found on https://jurassic.ninja/wp-json/jurassic.ninja/jetpack-beta/plugins/woocommerce-payments/branches - searching for the branch of this PR `rpp/6674-create-payment-service-class`. Ensure that the `commit` for this file is the same for the last commit in this PR. At the moment, it is `a026e9870e061fdaf8412f1c7755c9072b4abe8b`
- In the build, ensure the files in the new `src` folder are included. 

The build can be found on https://jurassic.ninja/wp-json/jurassic.ninja/jetpack-beta/plugins/woocommerce-payments/branches - searching for the branch of this PR `rpp/6674-create-payment-service-class`. 





*

-------------------


**Post merge**
